loops_cc.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#Use a for loop to print the numbers 1 to 20, inclusive
+for value in range(1,20):
```

Comment:
Style nit-pick; need a space after the comma: `range(1, 20)`.

Reply:
@paulghaddad Is the space a personal preference or best practice? The Crash Course book doesn't use spaces, which is why I didn't include them here.